content/Coalesce.Starter.Vue.Web/package.json

```diff
@@ -6,12 +6,13 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "test": "vue-cli-service test",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "coalesce": "dotnet coalesce ../coalesce.json"
```

Comment:
coalesce.json will be automatically found if it is named exactly that, so this can just be `"dotnet coalesce"`

Reply:
Excellent.